<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="6" skipped="0" tests="6" time="4.846" timestamp="2022-07-09T12:26:03.029579" hostname="LAPTOP-9RU984SS"><testcase classname="FYP.FYP.FYP.Test.test_Patient.test_patient" name="test_admin_login_with_default_password" file="FYP\FYP\FYP\Test\test_Patient.py" line="39" time="0.001"><failure message="AttributeError: 'test_patient' object has no attribute 'app'">self = &lt;FYP.Test.test_Patient.test_patient testMethod=test_admin_login_with_default_password&gt;

    def setUp(self):
       #self.app = Flask(__name__)
       #app.config['SECRET_KEY'] = 'facial_recognition'
       #app.config['MYSQL_HOST'] = 'localhost'
       #app.config['MYSQL_USER'] = 'root'
       #app.config['MYSQL_PASSWORD'] = ''
       #app.config['MYSQL_DB'] = 'healthcare_db'
       #app.config['TESTING'] = True
       #app.config['LOGIN_DISABLED'] = False
&gt;      self.client = self.app.test_client()
E      AttributeError: 'test_patient' object has no attribute 'app'

FYP\FYP\FYP\Test\test_Patient.py:19: AttributeError</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="FYP.FYP.FYP.Test.test_Patient.test_patient" name="test_index" file="FYP\FYP\FYP\Test\test_Patient.py" line="46" time="0.001"><failure message="AttributeError: 'test_patient' object has no attribute 'app'">self = &lt;FYP.Test.test_Patient.test_patient testMethod=test_index&gt;

    def setUp(self):
       #self.app = Flask(__name__)
       #app.config['SECRET_KEY'] = 'facial_recognition'
       #app.config['MYSQL_HOST'] = 'localhost'
       #app.config['MYSQL_USER'] = 'root'
       #app.config['MYSQL_PASSWORD'] = ''
       #app.config['MYSQL_DB'] = 'healthcare_db'
       #app.config['TESTING'] = True
       #app.config['LOGIN_DISABLED'] = False
&gt;      self.client = self.app.test_client()
E      AttributeError: 'test_patient' object has no attribute 'app'

FYP\FYP\FYP\Test\test_Patient.py:19: AttributeError</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="FYP.FYP.FYP.Test.test_Patient.test_patient" name="test_login" file="FYP\FYP\FYP\Test\test_Patient.py" line="29" time="0.001"><failure message="AttributeError: 'test_patient' object has no attribute 'app'">self = &lt;FYP.Test.test_Patient.test_patient testMethod=test_login&gt;

    def setUp(self):
       #self.app = Flask(__name__)
       #app.config['SECRET_KEY'] = 'facial_recognition'
       #app.config['MYSQL_HOST'] = 'localhost'
       #app.config['MYSQL_USER'] = 'root'
       #app.config['MYSQL_PASSWORD'] = ''
       #app.config['MYSQL_DB'] = 'healthcare_db'
       #app.config['TESTING'] = True
       #app.config['LOGIN_DISABLED'] = False
&gt;      self.client = self.app.test_client()
E      AttributeError: 'test_patient' object has no attribute 'app'

FYP\FYP\FYP\Test\test_Patient.py:19: AttributeError</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="FYP.FYP.FYP.Test.test_Patient.test_patient" name="test_pass_correct" file="FYP\FYP\FYP\Test\test_Patient.py" line="77" time="0.001"><failure message="AttributeError: 'test_patient' object has no attribute 'app'">self = &lt;FYP.Test.test_Patient.test_patient testMethod=test_pass_correct&gt;

    def setUp(self):
       #self.app = Flask(__name__)
       #app.config['SECRET_KEY'] = 'facial_recognition'
       #app.config['MYSQL_HOST'] = 'localhost'
       #app.config['MYSQL_USER'] = 'root'
       #app.config['MYSQL_PASSWORD'] = ''
       #app.config['MYSQL_DB'] = 'healthcare_db'
       #app.config['TESTING'] = True
       #app.config['LOGIN_DISABLED'] = False
&gt;      self.client = self.app.test_client()
E      AttributeError: 'test_patient' object has no attribute 'app'

FYP\FYP\FYP\Test\test_Patient.py:19: AttributeError</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="FYP.FYP.FYP.Test.test_Patient.test_patient" name="test_pass_incorrect" file="FYP\FYP\FYP\Test\test_Patient.py" line="85" time="0.001"><failure message="AttributeError: 'test_patient' object has no attribute 'app'">self = &lt;FYP.Test.test_Patient.test_patient testMethod=test_pass_incorrect&gt;

    def setUp(self):
       #self.app = Flask(__name__)
       #app.config['SECRET_KEY'] = 'facial_recognition'
       #app.config['MYSQL_HOST'] = 'localhost'
       #app.config['MYSQL_USER'] = 'root'
       #app.config['MYSQL_PASSWORD'] = ''
       #app.config['MYSQL_DB'] = 'healthcare_db'
       #app.config['TESTING'] = True
       #app.config['LOGIN_DISABLED'] = False
&gt;      self.client = self.app.test_client()
E      AttributeError: 'test_patient' object has no attribute 'app'

FYP\FYP\FYP\Test\test_Patient.py:19: AttributeError</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="FYP.FYP.FYP.Test.test_Patient.test_patient" name="test_users_login" file="FYP\FYP\FYP\Test\test_Patient.py" line="62" time="0.001"><failure message="AttributeError: 'test_patient' object has no attribute 'app'">self = &lt;FYP.Test.test_Patient.test_patient testMethod=test_users_login&gt;

    def setUp(self):
       #self.app = Flask(__name__)
       #app.config['SECRET_KEY'] = 'facial_recognition'
       #app.config['MYSQL_HOST'] = 'localhost'
       #app.config['MYSQL_USER'] = 'root'
       #app.config['MYSQL_PASSWORD'] = ''
       #app.config['MYSQL_DB'] = 'healthcare_db'
       #app.config['TESTING'] = True
       #app.config['LOGIN_DISABLED'] = False
&gt;      self.client = self.app.test_client()
E      AttributeError: 'test_patient' object has no attribute 'app'

FYP\FYP\FYP\Test\test_Patient.py:19: AttributeError</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase></testsuite></testsuites>